src/chronos_shared/ingestion_mode.py

```
@@ -0,0 +1,23 @@
+
+Placement rationale: ``mcp_gateway/upstream/context_store_client.py`` enforces
+"the gateway must NOT import anything from ``context_store``". Placing this
+SSOT under either subsystem would force a cross-package import. Hence the
+module lives in its own top-level package ``chronos_shared``.
+"""
+
+from __future__ import annotations
+
+from typing import Final, Literal
+
+__all__ = ["IngestionMode", "DEFAULT_INGESTION_MODE", "CHRONOS_INGESTION_MODE_ENV"]
```

Ruff (0.15.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)